lte/gateway/c/core/oai/include/service303_messages_types.h

```diff
@@ -33,4 +33,11 @@ typedef struct application_unhealthy_msg {
   uint8_t unused;
 } application_unhealthy_msg_t;
 
+// Message capturing stats as communicated by the mme_app
+typedef struct application_stats_msg {
```

Comment:
how about naming this `application_mme_stats_msg`?

Reply:
will do.